chart/githubconnector/values.yaml

```diff
@@ -0,0 +1,13 @@
+replicaCount: 1
```

Comment:
Great idea to move it to values, but you probably forgot to include it in deployment.